source/features/conversation-navigation.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import ChevronLeftIcon from 'octicon/chevron-left.svg';
+import ChevronRightIcon from 'octicon/chevron-right.svg';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import SearchQuery from '../github-helpers/search-query';
+import looseParseInt from '../helpers/loose-parse-int';
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+function getButton(direction: string): HTMLAnchorElement {
+	return (
+		<a
+			aria-disabled="true"
+			className="btn btn-lg BtnGroup-item"
+			aria-label={`Navigate to ${direction} Conversation`}
+			data-hotkey={direction.charAt(0)}
+		>
+			{direction === 'next' ? <ChevronRightIcon/> : <ChevronLeftIcon/>}
+		</a>
+	) as unknown as HTMLAnchorElement;
+}
+
+function setButtonHref(button: HTMLAnchorElement, query: string, page: number, conversation?: Conversation) {
+	if (conversation) {
+		const url = new URL(conversation.url);
+		url.searchParams.set('q', query);
+		url.searchParams.set('page', String(page));
+		button.removeAttribute('aria-disabled');
```

Comment:
Added the following will add the hovercard.
@fregante WDYT? I love it ❤️ 
```suggestion
		button.removeAttribute('aria-disabled');
		button.dataset.hovercardUrl = url.pathname + '/hovercard';
```